classes/fsl-eula-unpack.bbclass

```diff
@@ -171,14 +181,8 @@ python fsl_bin_do_unpack() {
             bb.fatal("%s: LIC_FILES_CHKSUM contains an invalid URL:  %s" % (d.getVar('PF'), license))
         if license == layer_license:
             bb.note("Found Freescale EULA for the layer %s." % license)
-            found_layer_license = True
+            found_layer_licenses += license
         elif parm.get('md5') in md5sums:
             bb.note("Found Freescale EULA for the package %s." % license)
-            found_package_license = True
-    if not found_layer_license:
-        bb.fatal("The Freescale layer EULA '%s' is not listed in LIC_FILES_CHKSUM '%s'."
-                 % (layer_license, licenses))
-    if not found_package_license:
-        bb.fatal("A valid package EULA with md5sum in %s was not found in LIC_FILES_CHKSUM '%s'."
-                 % (md5sums.split(), licenses))
-}
+            found_package_licenses += license
+    return (layer_license, licenses, md5sums, found_layer_licenses, found_package_licenses)
```

Comment:
only the first two arguments are used. It could reduce the number of tuple items.

Reply:
Not sure what's best. In my reuse case, I want the other values as well, so it does reduce a few lines of duplication.

Reply:
That's fine then.